UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -573,6 +588,36 @@ protected async Task _LoadScene(int sceneIndex = -1, bool showSceneObj = true, C
 			_lastLoadedScene = CreatedObject;
 		}
 
+		private void GetGtlfContentTotals(GLTFScene scene)
+		{
+			// Count Nodes
+			Queue<Node> nodeQueue = new Queue<Node>();
+
+			// Add scene nodes
+			for (int i = 0; i < scene.Nodes.Count; ++i)
```

Comment:
Probably want to check for null "Nodes" object

Reply:
Done